src/renderers/shared/fiber/ReactFiber.js

```diff
@@ -106,7 +109,7 @@ var createFiber = function(tag : number, key : null | string) : Fiber {
     memoizedInput: null,
     output: null,
 
-    hasPendingChanges: true,
+    pendingWorkPriority: 0,
```

Comment:
I suspect that we'll want to let this list grow - maybe even indefinitely. So that very low priority things can be internally prioritized. Such as individual list items.

`NoWork` could be `Infinity` but that's a float number so I don't want that. It could also be a very high number, but that means comparison is slower (marginally). It is also nice that it is falsy. Here and in C.